### PR TITLE
fix: small fixes

### PR DESCRIPTION
**Pull request by Daljit-fleetHawks**
## Change Description

Fixed some issues on maintenance components.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by amritfleethawks — APPROVED** [review]

File: src/app/pages/fleet/maintenance/service-log/service-detail/service-detail.component.html [hunk 1]
```diff
@@ -45,15 +45,15 @@
                                           </tr>
                                          <tr>
                                             <td class="back-color-gray font-weight-bold">Start Date</td>
-                                            <td>Jun 1, 2020</td>
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/maintenance/service-log/service-detail/service-detail.component.html:48]: Is this Canadian date format?

File: src/app/pages/fleet/maintenance/service-log/service-detail/service-detail.component.ts [hunk 2]
```diff
@@ -70,6 +71,7 @@ export class ServiceDetailComponent implements OnInit {
       error: () => {},
       next: (result: any) => {
         this.logsData = result.Items[0];
+        console.log('fdf', this.logsData)
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/maintenance/service-log/service-detail/service-detail.component.ts]: Remove this log

**Review by kunalfleethawks — APPROVED** [review]

**Review by Daljit-fleetHawks — COMMENTED** [review]
changes done

File: src/app/pages/fleet/maintenance/service-log/service-detail/service-detail.component.html (hunk 1 above)
Comment (Daljit-fleetHawks) [reply src/app/pages/fleet/maintenance/service-log/service-detail/service-detail.component.html:48]: I will changed

**Pull request merged**